### PR TITLE
DOC: fix typo in spatial join

### LINKED ISSUE #1187: Unclear "observations from to GeoSeries"
It seems that it should be "observations from GeoSeries"

at http://geopandas.org/mergingdata.html

fixed by #1188 (automatic linking by GH failed)

### PR DESCRIPTION
**Pull request by matkoniecz**
Fixes #1187

### CODE REVIEW

**Comment by martinfleis** [comment]
Hi @matkoniecz,

thank you for the fix. Since you started editing it, do you want to add this example as well to fill the gap you have just found https://github.com/geopandas/geopandas/issues/1186#issuecomment-548764634?

**Comment by matkoniecz** [comment]
Yes, I would be happy to send more PRs improving documentation (from a perspective of newbie).

**Comment by martinfleis** [comment]
Can we keep it within this PR so we don’t have many minor changes scattered in multiple PRs?
    	
    	

    	
    
  




On Fri, Nov 1, 2019 at 12:42 PM +0000, "Mateusz Konieczny" <notifications@github.com> wrote:












Yes, I would be happy to send more PRs improving documentation (from a perspective of newbie).

File: doc/source/mergingdata.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ There are two ways to combine datasets in *geopandas* -- attribute joins and spa
 
 In an attribute join, a ``GeoSeries`` or ``GeoDataFrame`` is combined with a regular *pandas* ``Series`` or ``DataFrame`` based on a common variable. This is analogous to normal merging or joining in *pandas*.
 
-In a Spatial Join, observations from to ``GeoSeries`` or ``GeoDataFrames`` are combined based on their spatial relationship to one another.
+In a Spatial Join, observations from ``GeoSeries`` or ``GeoDataFrames`` are combined based on their spatial relationship to one another.
```
Comment (jorisvandenbossche) [inline doc/source/mergingdata.rst]: I think the intent here was this to be "two". Would that also be OK?

**Comment by martinfleis** [comment]
Thank you @matkoniecz for noticing!

**Pull request merged**